src/basic/verbs.c

```diff
@@ -133,9 +141,28 @@ int dispatch_verb(int argc, char *argv[], const Verb verbs[], void *userdata) {
                         return r;
         }
 
-        if (name)
+        if (create_new_arg) {
+                char **fake;
+                int j;
+
+                fake = new0(char*, left + 1);
```

Comment:
hmm, given that this is pretty much bounded and the original array is essentially on the stack(-like) memory too, we might just use newa0() here...